lib/modules.nix

```diff
@@ -10,26 +10,20 @@ with lib.types;
 
 rec {
 
-  /* Evaluate a set of modules.  The result is a set of two
+  evalModules = import ./eval-modules.nix;
```

Comment:
This would equate the non-modular attributes allowed with that of `nixos/lib/eval-config.nix`. It should only allow the non-modular attributes of `evalModules` that were allowed before.